ioctl/util/util.go

```diff
@@ -90,17 +89,12 @@ func StringToRau(amount string, numDecimals int) (*big.Int, error) {
 
 // RauToString converts Rau big int into Iotx string
 func RauToString(amount *big.Int, numDecimals int) string {
-	var targetUnit int64
-	switch numDecimals {
-	case 18:
-		targetUnit = unit.Iotx
-	case 12:
-		targetUnit = unit.Qev
-	default:
-		targetUnit = unit.Rau
+	if numDecimals == 0 {
+		return amount.String()
 	}
+	targetUnit := new(big.Int).Exp(big.NewInt(10), big.NewInt(int64(numDecimals)), nil)
 	amountInt, amountDec := big.NewInt(0), big.NewInt(0)
-	amountInt.DivMod(amount, big.NewInt(targetUnit), amountDec)
+	amountInt.DivMod(amount, targetUnit, amountDec)
```

Comment:
30 lines below
return RauToString(amountInt, 18), nil
change 18 to IotxDecimalNum